app/lib/vendor_api/changes/notes_for_application.rb

```diff
@@ -0,0 +1,12 @@
+module VendorAPI
+  module Changes
+    class NotesForApplication < VersionChange
+      description 'Includes notes associated with the application'
+
+      action ApplicationsController, :show
```

Comment:
```suggestion
```

Reply:
Removing this breaks any subsequent version change class configuration, because the `ValidVendorApiRoute` expects an action here https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/initializers/route_validators.rb#L30